jsx/Form.js

```diff
@@ -469,6 +469,38 @@ class SelectElement extends Component {
     this.handleChange = this.handleChange.bind(this);
   }
 
+  /**
+   * @componentDidMount
+   *
+   * Call onUserInput on component rendered to select only option
+   * if autoSelect prop is set to true
+   */
+  componentDidMount() {
+    const optionsArray = Object.keys(this.props.options);
+    if (this.props.autoSelect && optionsArray.length === 1) {
+      this.props.onUserInput(this.props.name, optionsArray[0]);
+    }
+  }
+
+  /**
+   * @componentDidUpdate
+   * @param {object} prevProps - component props before component update
```

Comment:
This documentation style is weird.. the description goes above the parameters in the rest of our documentation (also has the weird tag..)

Reply:
yeahh this is a new requirement from eslinst and i just followed the syntax that already existed elsewhere in the file. i can modify the comment to be in whatever way, but i don't know what the proper way should be for our JS documentation. i don't think we have a standard yet?

Reply:
you're right that we don't have a standard but the rest of the file goes:

```
Description

@param {type} argname - description
@return {type} 
```

from what I see? (Which is more consistent with our PHP documentation..)

Reply:
The only place there's other `@something` is when it's `@constructor` on constructors instead of a description

Reply:
truee i'll change it to suit that